zone/mob.cpp

```diff
@@ -4057,37 +4057,6 @@ void Mob::SetNextIncHPEvent( int inchpevent )
 	nextinchpevent = inchpevent;
 }
 
```

Comment:
Any reason this was removed? Was it just not used anywhere?

Reply:
Believe you meant ```int16 Mob::GetResist(uint8 type) const``` but, yes removed as it's not used, was replaced by ```int Mob::GetResist(uint8 resist_type)``` that contains all of the newer resist types.